### PR TITLE
Fix invalid SASS syntax, fixes #443

### LINKED ISSUE #443: Remove closing semicolon
1. Can you please remove the closing semicolon on this line and the line below: https://github.com/jgthms/bulma/blob/master/sass/utilities/functions.sass#L32
It throws a compiler error.

2. Can you move the single-line comments on this line and the line below: https://github.com/jgthms/bulma/blob/master/sass/utilities/mixins.sass#L176
Single-line comments must be on their own line in the indented syntax.

Thx!

### PR DESCRIPTION
**Pull request by werthen**
This blocks compilation for certain users, and is invalid sass syntax

Changes proposed:

- [ ] Add
- [x] Fix
- [ ] Remove
- [ ] Update

### CODE REVIEW

File: sass/utilities/mixins.sass [hunk 1]
```diff
@@ -173,8 +173,10 @@
 // Responsiveness
 
 $tablet: 769px !default
-$desktop: 1000px !default // 960px container + 40px
-$widescreen: 1192px !default // 1152px container + 40
+// 960px container + 40px
+$desktop: 1000px !default
+// 1152px container + 40
+$widescreen: 1192px !default
```
Comment (jgthms) [inline sass/utilities/mixins.sass]: Why this change?

Comment (werthen) [inline sass/utilities/mixins.sass]: Inline comments do not work in the indented syntax, they should be on their own line.

Comment (werthen) [inline sass/utilities/mixins.sass]: Oh actually, looking into it further, it seems to choke after `!default` specifically https://github.com/sass/sass/issues/326#issuecomment-5272861

Comment (janwh) [inline sass/utilities/mixins.sass]: Googling this very issue I'm finally arriving here. I would greatly appreciate @jgthms merging this PR.

Comment (BrunoViera) [inline sass/utilities/mixins.sass]: This is the error that I have with the comments in the same line that the variable definition

image
